packages/types/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './constants';
 export * from './models';
+export * from './models/V1beta1ProviderStatus';
```

Comment:
this should be covered by `export * from './models'` if you need it, you may have a problem somewhere else, maybe missing it in `./models/index.ts` or some import loop ?

Reply:
please find the reason for this, and remove this unneeded line

Reply:
we can find a fix in a follow up